kmip/tests/unit/core/attributes/test_digest.py

```diff
@@ -316,9 +318,17 @@ def test_not_equal_on_not_equal(self):
             digest_value=self.digest_value_b,
             key_format_type=self.key_format_type_b)
         b = Digest()
+        c = Digest(
+            hashing_algorithm=self.hashing_algorithm_d,
+            digest_value=self.digest_value_b,
+            key_format_type=self.key_format_type_c)
+        d = Digest(
+            key_format_type=self.key_format_type_c)
 
         self.assertTrue(a != b)
         self.assertTrue(b != a)
+        self.assertTrue(b != c)
+        self.assertTrue(b != d)
```

Comment:
I suggest modifying `__eq__` to resemble the following:
```
            return (self.hashing_algorithm == other.hashing_algorithm and
                    self.digest_value == other.digest_value and
                    self.key_format_type == other.key_format_type)
```
which is shorter, clearer, and allows a single comparison of two equal objects to cover all branches and statements.